File: src/main/java/gargoyle/ct/prop/impl/ro/CTReadOnlyByteProperty.java
package gargoyle.ct.prop.impl.ro;

import gargoyle.ct.prop.CTReadOnlyNumberProperty;
import org.jetbrains.annotations.NotNull;

public class CTReadOnlyByteProperty extends CTBaseReadOnlyProperty<Byte> implements CTReadOnlyNumberProperty<Byte> {
    public CTReadOnlyByteProperty(String name, @NotNull Byte value) {
        super(name, value);
    }
}

File: util/src/main/kotlin/gargoyle/ct/util/prop/impl/ro/CTReadOnlyByteProperty.kt
package gargoyle.ct.util.prop.impl.ro

import gargoyle.ct.util.prop.CTReadOnlyNumberProperty


class CTReadOnlyByteProperty(name: String, value: Byte) : CTBaseReadOnlyProperty<Byte>(name, value),
    CTReadOnlyNumberProperty<Byte>
